hypha/static_src/sass/components/_form.scss

```diff
@@ -38,13 +38,28 @@
             padding-left: 0.5rem;
             position: relative;
             max-width: 100%;
-
             &::before {
                 display: none;
             }
         }
     }
 
+    &--width {
+        flex-grow: 2;
+    }
+
+    &--score-box {
+        display: none;
+        background-color: #1d79a8;
+        color: #fff;
+        padding: 0.5em 60px;
```

Comment:
Please use `rem` for all paddings.

Reply:
We also mostly use tailwind classes for new styling nowadays. But if a component already exists like here it is ok to build on that as well.